source/process/community-overview.rst

```diff
@@ -155,6 +155,13 @@ Recognizing people's contributions is vital to keeping them engaged. Leverage th
 
 When possible, influence contributors to apply themselves to the highest impact tickets and projects appropriate for their skill level. Meaningful contributions early in the contributor journey are a powerful way to draw people into the community. 
 
+Community Moderators / Community Support Champions
```

Comment:
Thanks Amy! Reading some of other categories, wondering if this overlaps with knowledge base contributors and should be merged with that one? 

https://github.com/mattermost/docs/blob/cf5c8e68bcd44a56c0d3f6213d493523a847c675/source/process/community-overview.rst#knowledge-base-contributors

Both yours and that section have good items to include in the definition.